### PR TITLE
fix(marshaller): WU Results should share WU instances

### PR DESCRIPTION
**Pull request by GordonSmith**
Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

File: packages/marshaller/src/ddl2/dashboard.ts [hunk 1]
```diff
@@ -139,3 +140,108 @@ export class Dashboard extends DockPanel implements IClosable {
     }
 }
 Dashboard.prototype._class += " dashboard_dashboard";
+
+export class DashboardCP extends ChartPanel {
+
+    private _ec: ElementContainer;
+
+    private _addButton = new Button("fa-plus", "Add...")
```
Comment (jbrundage) [inline packages/marshaller/src/ddl2/dashboard.ts]: This will break when #2998 is pulled

File: packages/marshaller/src/ddl2/dashboard.ts [hunk 2]
```diff
@@ -139,3 +140,108 @@ export class Dashboard extends DockPanel implements IClosable {
...
+            }
+        });
+
+    private _removeButton = new Button("fa-minus", "Remove...")
+        .enabled(false)
+        .on("click", () => {
+            this.remove();
+        });
+
+    private _addSamples = new Button("fa-database", "Add Samples")
```
Comment (jbrundage) [inline packages/marshaller/src/ddl2/dashboard.ts]: This will break when #2998 is pulled

File: packages/marshaller/src/ddl2/dashboard.ts [hunk 3]
```diff
@@ -139,3 +140,108 @@ export class Dashboard extends DockPanel implements IClosable {
...
+            if (this._contextMenu.open({
+                target: node,
+                currentTarget: node,
+                clientX: rect.left,
+                clientY: rect.bottom
+            })) {
+            }
+        });
+
+    private _removeButton = new Button("fa-minus", "Remove...")
```
Comment (jbrundage) [inline packages/marshaller/src/ddl2/dashboard.ts]: This will break when #2998 is pulled

**Comment by GordonSmith** [comment]
I will need to update the other PR (or this one depending on which got pulled first)

**Pull request merged**